conf/tax_years.conf

```diff
@@ -17,6 +17,24 @@ latestTaxYear.enabled = true
 # IMPORTANT: Please add new tax year at top of this list.
 #########################################
 tax-years = [
+  {
+    start-year = 2022
```

Comment:
Have you checked these values with Sujeet

Reply:
I had a call with Sujeet, he said that all of the value are the same as last year.